src/java.desktop/share/classes/java/awt/Graphics2D.java

```diff
@@ -1200,10 +1200,14 @@ public abstract boolean hit(Rectangle rect,
      * previously been set, or if the clip has been cleared using
      * {@link Graphics#setClip(Shape) setClip} with a {@code null}
      * argument, the specified {@code Shape} becomes the new
-     * user clip.
+     * user clip. Since this method intersects the specified shape
+     * with the current clip, it will throw NPE for a null shape
```

Comment:
Shall NPE be spelled out as `{@code NullPointerException}`?

Also should it rather be, “…it <del>will</del> throws NPE…”?

Reply:
Done..As for the statement change, I think "will throw" is more grammatically correct" so kept it unchanged.